bin/cml.test.js

```diff
@@ -8,23 +8,26 @@ describe('command-line interface tests', () => {
       "cml.js <command>
 
       Commands:
-        cml.js ci                                 Fixes specific CI setups
-        cml.js pr <glob path...>                  Create a pull request with the
-                                                  specified files
-        cml.js publish <asset>                    Upload an image to build a report
-        cml.js rerun-workflow                     Reruns a workflow given the jobId or
-                                                  workflow Id
-        cml.js runner                             Launch and register a self-hosted
-                                                  runner
-        cml.js send-comment <markdown file>       Comment on a commit
-        cml.js send-github-check <markdown file>  Create a check report
-        cml.js tensorboard-dev                    Get a tensorboard link
+        cml.js check              Manage CI checks
+        cml.js comment            Manage comments
+        cml.js pr <glob path...>  Manage pull requests
+        cml.js runner             Manage self-hosted (cloud & on-premise) CI runners
+        cml.js tensorboard        Manage tensorboard.dev connections
+        cml.js workflow           Manage continuous integration workflows
+        cml.js ci                 Prepare Git repository for CML operations
+
+      Global Options:
+        --log     Maximum log level
+                [string] [choices: \\"error\\", \\"warn\\", \\"info\\", \\"debug\\"] [default: \\"info\\"]
+        --driver  Git provider where the repository is hosted
+          [string] [choices: \\"github\\", \\"gitlab\\", \\"bitbucket\\"] [default: infer from the
+                                                                          environment]
+        --repo    Repository URL or slug[string] [default: infer from the environment]
+        --token   Personal access token [string] [default: infer from the environment]
 
       Options:
         --help     Show help                                                 [boolean]
```

Comment:
Also, is it possible to add a `-h` alias for `--help`?

Reply:
> _shouldn't `--help` be in Global Options?_

Given that `--help` is local to child commands, it probably shouldn't.[^1]

[^1]: Also being lazy because doing such a small change requires an ugly technical workaround.

Reply:
> _Also, is it possible to add a -h alias for --help?_

Yes, it's possible, although, given that no other option has a single-character shorthand and — as any other unknown option — it displays the help message (e.g. `cml -h` behaves similarly to `cml -xyz` and `cml --help`), I would suggest leaving it as is.

Feel free to resume this thread if you feel it would be useful to add a dedicated alias.

Reply:
Nevermind, resolved with https://github.com/iterative/cml/pull/1073/commits/e1f4651c26d8d47b58eed2ae8fe2228973c9bd5f 🤢